packages/modelence/src/app/index.ts

```diff
@@ -145,6 +145,10 @@ function getRateLimits(modules: Module[]) {
 function getConfigSchema(modules: Module[]): ConfigSchema {
   const merged: ConfigSchema = {};
 
+  for (const [key, value] of Object.entries(systemConfigSchema)) {
+    merged[key] = value;
+  }
+
```

Comment:
`modules` passed to this function already include system modules, so this doesn't seem right - I think we should derive system config schema the same way. The current problem is that system modules are simply not defining their own config schema as needed.